docusaurus.config.js

```diff
@@ -8,20 +8,23 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 function getFrontmatter(field) {
   try {
-    const readme = read('./README.md', { excerpt: true });
+    const readme = read('/app/docs/README.md', { excerpt: true });
     if (typeof readme.data[field] !== 'undefined') {
       console.log(`Using ${readme.data[field]} as website ${field}`)
       return readme.data[field]
     }
-    console.log(`Cannot find ${field} in README.md, using default value`)
+    else {
+      console.log(`Cannot find ${field} in README.md, using default value - ${process.env.PROJECT_NAME}`)
+      return process.env.PROJECT_NAME
+    }
   } catch (error) {
     throw error
   }
 }
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: getFrontmatter('project_name') || process.env.PROJECT_NAME || "",
```

Comment:
Hey @Bucknalla can you explain why we are removing the default? I would say we should have something like this instead. This change would be breaking project builds all over the place for those that don't have a frontmatter and title can't be empty

`getFrontmatter('project_name') || process.env.PROJECT_NAME`

Reply:
I see now what you did. But, it makes the function too specific to only be used for the README's frontmatter capable of only returning project_name. Sooner than later we would hit another wall. Making an improvement.

Reply:
Opened #38